src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -158,7 +167,7 @@ private static RepoConfiguration getMatchingRepoConfig(
     /**
      * Returns a list of {@link RepoConfiguration} where the {@link RepoLocation} matches {@code targetRepoLocation}.
      */
-    private static List<RepoConfiguration> getMatchingRepoConfigsByRepoLocation(
+    private static List<RepoConfiguration> getMatchingRepoConfigsByLocation(
```

Comment:
Is this change necessary?

Reply:
This was changed according to fzdy1914's review.
> Suggest to be `getMatchingRepoConfigsByLocation` for simplisity.

Reply:
I thought this is a newly added method previously so I made that comment. But anyway I believe the current one is more clear.